apps/cowswap-frontend/src/common/hooks/useNavigate.ts

```diff
@@ -0,0 +1,17 @@
+import { isInjectedWidget } from '@cowprotocol/common-utils'
```

Comment:
this file is the gist of this PR. doesn't do much, just delegate to the useNavigate setting some nice defaults. It still allows you to override the defaults though